python/pyfeelpp-mor/feelpp/mor/nirb/test_process.py

```diff
@@ -0,0 +1,85 @@
+#%%
+## Import module 
+from postProcess import * 
+
+#%%
+
+if __name__ == "__main__":
+    
+    import sys 
+
+    dataPath = "/data/home/elarif/feelppdb/nirb/heat/np_1/results/Rect/"
```

Comment:
this should not be hardcoded, please use feel++ framework and repositories

Reply:
we replace this file by a notebook one in the branch [#2041](https://github.com/feelpp/feelpp/issues/2041). This will be easy to manipulate.